recipes/yamlloader/meta.yaml

```diff
@@ -0,0 +1,47 @@
+#{% set name = "yamlloader" %}
```

Comment:
Looks like this `#` snuck in. Fixing with PR ( https://github.com/conda-forge/staged-recipes/pull/5870 ).

Reply:
Oops, don't know how that happened. Thanks!